.dockerignore

```diff
@@ -0,0 +1,7 @@
+.git
+.github
+.gitignore
+.cache
+1.14.5/
+tests/
+tools/
```

Comment:
please add newline

Reply:
hmm i added a newline on both and for some reason they're not reflecting on either?